controllers/home.js

```diff
@@ -10,6 +10,9 @@ function home(req, res) {
   if (req.query.breach) {
     const reqBreachName = req.query.breach.toLowerCase();
     featuredBreach = req.app.locals.breaches.filter(breach => breach.Name.toLowerCase() === reqBreachName)[0];
+    if (!featuredBreach) {
+      throw new FluentError("home-not-found");
```

Comment:
`throw` makes this return with `500` status code.

We should try to change this to do like the `notFound` route handler does: set `res.status(404)` and render the error subpage with the right message.

But I'm not 100% sure if that's easy, and this would be better than what we have now. So if we can't figure it out quickly, I'm okay to merge this and leave the `404` status code as a follow-up.

Reply:
This isn't as simple as it should be, but I'm going to go ahead and merge so we can land the half-fix.